src/main-window.ui

```diff
@@ -23,7 +23,7 @@
     <item>
      <widget class="QSplitter" name="mainWinSplitter">
       <property name="orientation">
-       <enum>Qt::Vertical</enum>
+       <enum>Qt::Horizontal</enum>
```

Comment:
Changing the default for debugging or development is okay during development, but those changes should not get merged. The default orientation is still `Qt::Vertical` so the .ui file can still easily be edited with Qt Designer.